### PR TITLE
ADD contract_account_banking_mandate module

### PR DESCRIPTION
**Pull request by elacunza**
To define the banking mandate for recurring invoices

### CODE REVIEW

File: contract_account_banking_mandate/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Contract Account Banking Mandate
```
Comment (pedrobaeza) [inline contract_account_banking_mandate/README.rst:6]: Use new README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: contract_account_banking_mandate/README.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+================================
+
+This module adds an account banking mandate field to contracts. This is used
+for the creation of recurring invoices.
+
+Known issues / Roadmap
+======================
+
+ * no known issues
```
Comment (pedrobaeza) [inline contract_account_banking_mandate/README.rst]: Remove empty sections.

File: contract_account_banking_mandate/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Binovo IT Human Project SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from . import models
```
Comment (pedrobaeza) [inline contract_account_banking_mandate/__init__.py]: Line at the end

File: contract_account_banking_mandate/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "author": "Binovo IT Human Project SL, Odoo Community Association (OCA)",
+    "category": 'Sales Management',
+    "website": "http://www.binovo.es",
+    "summary": "Add banking mandate field to contracts for their invoices",
+    "data": [
+        "views/contract_view.xml",
+    ],
+    "installable": True,
+    "auto_install": False,
+}
```
Comment (pedrobaeza) [inline contract_account_banking_mandate/__openerp__.py]: Line at the end

File: contract_account_banking_mandate/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Binovo IT Human Project SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Contract Account Banking Mandate",
+    "version": "1.0.0",
```
Comment (pedrobaeza) [inline contract_account_banking_mandate/__openerp__.py]: 8.0.1.0.0

File: contract_account_banking_mandate/models/__init__.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Binovo IT Human Project SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from . import account_analytic_account
+
```
Comment (pedrobaeza) [inline contract_account_banking_mandate/models/__init__.py]: Extra line to remove

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by elacunza** [comment]
Sorry for the noise, this was for 8.0 branch, please see #38

**Pull request closed without merging**